packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx

```diff
@@ -70,14 +93,20 @@ const PlatformDropdown = ({ app_routing_history, closeDrawer, platform_config })
 
     const platform_dropdown = (
         <div className='platform-dropdown'>
-            <Div100vhContainer className='platform-dropdown__list' height_offset='156px' is_disabled={isDesktop()}>
-                {platform_config.map(platform => {
-                    return (
-                        <div key={platform.name} onClick={closeDrawer} ref={ref}>
-                            <PlatformDropdownContent platform={platform} app_routing_history={app_routing_history} />
-                        </div>
-                    );
-                })}
+            <Div100vhContainer height_offset='9rem' is_disabled={isDesktop()}>
+                <div className='platform-dropdown__list'>
```

Comment:
This change removes scroll from platform switcher in responsive view, please check